### PR TITLE
Issue5317

### LINKED ISSUE #5317: Contributing guide: Internationalizing strings in JavaScript code section misleading about _() and ngettext()
### CKAN 2.8

The first sentence of [section _Internationalizing strings in JavaScript code_ of the Contributing guide](https://docs.ckan.org/en/2.8/contributing/string-i18n.html#internationalizing-strings-in-javascript-code) is:
> Each CKAN JavaScript module offers the methods _ and ngettext for translating singular and plural strings, respectively: [...]

It is correct that ngettext() can be used for plural strings, but the implication that ngettext() cannot translate singular strings and the implication that \_() can only translate singular strings are wrong. The sentence should presumably distinguish between invariant strings (_()) and number-inflected strings (ngettext()).

### PR DESCRIPTION
**Pull request by Gauravp-NEC**
Fixes #5317 

### Features:

- [ ] includes tests covering changes
- [ ] includes updated documentation
- [ ] includes user-visible changes
- [ ] includes API changes
- [ ] includes bugfix for possible backport

Please [X] all the boxes above that apply

### CODE REVIEW

**Comment by Gauravp-NEC** [comment]
I have update the document of String internationalization (https://docs.ckan.org/en/2.8/contributing/string-i18n.html#internationalizing-strings-in-javascript-code) in the commit  https://github.com/ckan/ckan/pull/5696/commits/c23967a7a988931d91b1e7b471e038e4b907b81c. Please ignore commit https://github.com/ckan/ckan/pull/5696/commits/e92a319b5991db72ba72e9ce064f964977c463ed and https://github.com/ckan/ckan/pull/5696/commits/d68c19bd048db579a65d53a791db0c98886af3b0 by mistake commits have been made.

**Comment by Gauravp-NEC** [comment]
@amercader, In this PR, commit https://github.com/ckan/ckan/commit/c23967a7a988931d91b1e7b471e038e4b907b81c (updated string-i18n.rst) is the relevant commit which may fix the issue https://github.com/ckan/ckan/issues/5317. By mistake commits https://github.com/ckan/ckan/commit/e92a319b5991db72ba72e9ce064f964977c463ed and https://github.com/ckan/ckan/commit/d68c19bd048db579a65d53a791db0c98886af3b0 have been made by me which are useless and not related to issue https://github.com/ckan/ckan/issues/5317. Can I raise a new PR to avoid any conflict? I am waiting for your response.

File: doc/contributing/string-i18n.rst [hunk 1]
```diff
@@ -206,8 +206,8 @@ Internationalizing strings in JavaScript code
 ---------------------------------------------
 
 Each :ref:`CKAN JavaScript module <javascript_modules>` offers the methods
-``_`` and ``ngettext`` for translating singular and plural strings,
-respectively:
+``_`` and ``ngettext``. The ``ngettext`` is used to translate a single string which
+has both a singular and a plural form. Whereas ``_``is used to translate a single string only:
```
Comment (amercader) [inline doc/contributing/string-i18n.rst:210]:
```suggestion
``_`` and ``ngettext``. The ``ngettext`` function is used to translate a single string which
has both a singular and a plural form, whereas ``_`` is used to translate a single string only:
```

**Comment by amercader** [comment]
@Gauravp-NEC better to create a fresh PR to avoid problems. I also added a minor suggestion. Thanks!

**Comment by Gauravp-NEC** [comment]
@amercader , Thankyou for your response. I have raise a fresh PR https://github.com/ckan/ckan/pull/5701 to fix the issue https://github.com/ckan/ckan/issues/5317

**Comment by Gauravp-NEC** [comment]
@amercader , Thankyou for your response. I have raise a fresh PR.

**Pull request closed without merging**